Frontend/src/components/TaskList.tsx

```diff
@@ -0,0 +1,61 @@
+import { useState, useEffect } from 'react';
+import axios from 'axios';
+import CsrfToken from './CsrfToken';
+
+export interface ITask {
```

Comment:
Good job with creating the interface and good naming! It's exactly as it should be!